multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -56,6 +76,29 @@ coreos:
         [Install]
         WantedBy=multi-user.target
 
+{{ if eq .ContainerRuntime "rkt" }}
+    - name: rkt-api.service
+      command: start
+      enable: true
+      content: |
+        [Service]
+        ExecStart=/usr/bin/rkt api-service
+        Restart=always
+        RestartSec=10
+
+    - name: load-rkt-stage1.service
+      command: start
+      content: |
+        [Unit]
+        Description=Load rkt stage1 images
+        Documentation=http://github.com/coreos/rkt
+        Requires=network-online.target
+        After=network-online.target
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/bin/rkt fetch /usr/lib/rkt/stage1-images/stage1-coreos.aci /usr/lib/rkt/stage1-images/stage1-fly.aci  --insecure-options=image
```

Comment:
I'm not sure what flag it used to be, euank mentioned the flag that let you specify an on-disk stage one is no longer around. It might just be the case that rather, it is not longer supported.

I just checked this repo's history and at one point I supply an on-disk stage one using the normal flag we use now, but it appears to no longer work with on-disk stage1 images. It is rejected for not being a valid ACI identifier. Not sure if I ever had that working on an older version of kubernetes or not.